src/js/notifications.js

```diff
@@ -0,0 +1,67 @@
+var Wand = (function(wand) {
+
+/* Inspiration taken from
+   https://github.com/cugos/dropchop/blob/master/src/js/notifications/Notifications.js
+ */
+
+  'use strict';
+
+  wand = wand || {};
+  wand.notifications = {};
+  var params;
+
+  wand.notifications.init = function() {
```

Comment:
:+1: I think the only other place that needs to be broken out is with the state rendering bit.